dataflowengineoss/src/main/scala/io/joern/dataflowengineoss/queryengine/SourcesToStartingPoints.scala

```diff
@@ -147,7 +147,12 @@ class SourceToStartingPointsInMethod(
   private def usageInOtherClasses(m: Method, usageInputs: List[UsageInput]): List[StartingPointWithSource] = {
     usageInputs.flatMap { case UsageInput(src, typeDecl, astNode) =>
       m.fieldAccess
-        .where(_.argument(1).isIdentifier.typeFullNameExact(typeDecl.fullName))
+        .where(
+          _.or(
+            _.argument(1).isIdentifier.typeFullNameExact(typeDecl.fullName),
+            _.argument(1).isTypeRef.typeFullNameExact(typeDecl.fullName)
+          )
+        )
```

Comment:
slight nit-pick: we can simplify `.where(_.or(...))` to just `.or(...)`
```suggestion
        .or(
          _.argument(1).isIdentifier.typeFullNameExact(typeDecl.fullName),
          _.argument(1).isTypeRef.typeFullNameExact(typeDecl.fullName)
        )
```

Reply:
Thanks! Good to know!